### PR TITLE
Update profiler docs

### LINKED ISSUE #51: Update profiler documentation
It was taken from the old Top Queries docs, but needs to be refreshed.

### PR DESCRIPTION
**Pull request by Andertech**
This is a WIP; fixes #51

### CODE REVIEW

File: content/how-to-use-vividcortex/profiler.md [hunk 1]
```diff
@@ -20,10 +20,7 @@ By default, the Profiler groups similar queries together, and shows the total ac
...
 3. Selecting a query inverts its color and displays details about it in the information pane. Hovering over a cell pops up an info tooltip.
-4. The query's abstracted form is shown in the information pane for easy examination and copying.
-5. Queries may have notifications of important information. Clicking on a query with a notification will reveal details in the sidebar.
-6. Hover over the per-host breakdown to see how the rank-by metric is distributed amongst the selected hosts.
-7. Sparklines of key performance metrics show you at a glance how average latency, total accumulated time, and execution count vary over time.
+5. Queries may have notifications of important information. Clicking on a query with a notification will navigate to the query details page.
```
Comment (gkristic) [inline content/how-to-use-vividcortex/profiler.md]: Number 4?

File: content/how-to-use-vividcortex/profiler.md [hunk 2]
```diff
@@ -20,10 +20,7 @@ By default, the Profiler groups similar queries together, and shows the total ac
   * You can choose the number of queries to show, and filter queries by text contained within them.
 2. The main table of results is sortable by clicking on the column headers. The main column displays a sparkline of the metric by which you ranked (VividCortex captures 1-second resolution for query metrics, as usual). In the example, the sparkline shows how the query's total accumulated time varies during the selected time range. You can hover over the sparklines to reveal the values at the desired point in time. Additional columns are shown to the right. Columns contain values and blue shaded backgrounds. The shaded area represents the proportion of that column's sum; hover over a cell for details.
 3. Selecting a query inverts its color and displays details about it in the information pane. Hovering over a cell pops up an info tooltip.
```
Comment (gkristic) [inline content/how-to-use-vividcortex/profiler.md]: I can't select the query. If I click, I go to the query page. If I only hover, the query doesn't get selected. Speaking of hovering, "Hovering over a cell pops up an info tooltip" should be "Hovering over the query..." I don't see any tooltip if I hover over the count or average latency cells.

File: content/how-to-use-vividcortex/profiler.md (hunk 1 above)
Comment (gkristic) [inline content/how-to-use-vividcortex/profiler.md]: Clicking on *any* query will navigate to the query details page. Like this it seems like only those with notifications.

File: content/how-to-use-vividcortex/profiler.md [hunk 3]
```diff
@@ -41,28 +38,27 @@ options panel at the top:
...
 
-![Profiler](/img/docs/top-queries-samples.png)
+![Profiler](/img/docs/query-detail-samples.png)
 
-When you click on the Samples link to drill down into the selected query, the query's details expand to fill most of the page, and the selected row from the original Profiler table stays at the top of the page. The center of the page now shows individual examples of queries in a scatterplot over time, with the vertical axis representing the query's execution time. Clicking on dots selects the corresponding sample and updates the summary to the left of the scatterplot. The selected query's dot turns gray, as shown by the dot near the red circled number 1.
+When you click on the Samples tab to drill down into the selected query, the selected row from the original Profiler table stays at the top of the page. The page now shows individual examples of queries in a scatterplot over time, with the vertical axis representing the query's execution time. Clicking on dots selects the corresponding sample and updates the summary at the bottom of the scatterplot. The selected query's dot turns gray.
```
Comment (gkristic) [inline content/how-to-use-vividcortex/profiler.md]: "... the selected row from the original Profiler table stays at the top of the page." I wouldn't say that the *row* stays at the top. A row is defined only as part of a table, but there's no table at the top of this page. I'd say that the *query* selected at the Profiler table will be displayed at the top, instead.

File: content/how-to-use-vividcortex/profiler.md [hunk 4]
```diff
@@ -41,28 +38,27 @@ options panel at the top:
...
 
 When selected, a sample's exact query text is shown below, with additional
 information as available, such as execution plan (EXPLAIN plan).
 
 ## Viewing Query Notifications
 
-Because VividCortex captures the network traffic to the database server, we are able to observe queries that return errors or warnings to client applications. These could be anything from permission errors to lock wait timeouts or syntax errors. Each type of error or warning will result in a notification in the main table, and when selected, in the sidebar:
+Because VividCortex captures the network traffic to the database server, we are able to observe queries that return errors or warnings to client applications. These could be anything from permission errors to lock wait timeouts or syntax errors. Each type of error or warning will result in a notification in the main table, and when selected, in the query details page:
```
Comment (gkristic) [inline content/how-to-use-vividcortex/profiler.md]: "... when selected, in the query details page". You don't actually *select* a page, right? You navigate to it or you don't; the page is displayed or it's not. I'd say "... will result in a notification both in the main Profiler table and in the query details page". It's clear that to see the notification in the query details page, you need to go to that page, much like you need to go to the Profiler tool to see the main table and hence the warning indication there.

**Comment by gkristic** [comment]
Other than that... :beers:

**Comment by Andertech** [comment]
Thanks @gkristic !!

**Comment by Andertech** [comment]
I think this is done @gkristic @mtrias ... It can and should be improved, since the screenshots I took are  unrelated  to one another. But otherwise I think it can be shipped in the meantime.

File: content/how-to-use-vividcortex/profiler.md [hunk 5]
```diff
@@ -12,18 +12,15 @@ The Profiler tool shows queries that execute on the selected systems during the
...
-3. Selecting a query inverts its color and displays details about it in the information pane. Hovering over a cell pops up an info tooltip.
-4. The query's abstracted form is shown in the information pane for easy examination and copying.
-5. Queries may have notifications of important information. Clicking on a query with a notification will reveal details in the sidebar.
-6. Hover over the per-host breakdown to see how the rank-by metric is distributed amongst the selected hosts.
-7. Sparklines of key performance metrics show you at a glance how average latency, total accumulated time, and execution count vary over time.
+3. Hovering over a query highlights its row in the table. If you click on the query it will navigate you to its query details page.
+4. Queries may have notifications of important information. Clicking on any query will navigate to the query details page.
```
Comment (mtrias) [inline content/how-to-use-vividcortex/profiler.md]: There's a bit of duplication between points 3 and 4. I think we should change "Clicking on any query will navigate to the query details page" in point 4, to "Navigate to the Query Details to see additional information about these notifications"

**Comment by mtrias** [comment]
just one minor comment. :ship:

**Comment by Andertech** [comment]
Got you covered @mtrias, thanks for reviewing! :ship: ing

**Pull request merged**